outlook/calendar/pkg/recurrence/recurrence_test.go

```diff
@@ -0,0 +1,317 @@
+package recurrence
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TODO - test these for real with Outlook
+func TestRecurrenceGeneration(t *testing.T) {
```

Comment:
I didn't set up any CI for these tests, but running them locally, they all pass. These test to make sure that the internal tool call to generate the recurrence object works the way we expect.